docs_src/examples/Ch-ExamplesAddingModbusDevice.md

```diff
@@ -340,7 +340,7 @@ File = ''
 
 Because we deploy EdgeX using docker-compose, we must add the
 device-modbus to the docker-compose file (
-<https://github.com/edgexfoundry/developer-scripts/blob/master/releases/geneva/compose-files/docker-compose-geneva-redis.yml>
+<https://github.com/edgexfoundry/edgex-compose/blob/geneva/docker-compose-geneva-redis.yml>
```

Comment:
Note that this link will be correct once PR for Geneva branch is merged. Updating this example to Hanoi is out of scope.